src/views/charts/TradingRewardsChart.tsx

```diff
@@ -88,8 +89,27 @@ export const TradingRewardsChart = ({
     abacusStateManager.setHistoricalTradingRewardPeriod(HistoricalTradingRewardsPeriod.DAILY);
   }, [canViewAccount]);
 
+  const rewardsData = useMemo(
+    () =>
+      periodTradingRewards && canViewAccount
+        ? periodTradingRewards
+            .toArray()
+            .reverse()
+            .map(
+              (datum) =>
+                ({
+                  date: new Date(datum.endedAtInMilliseconds).valueOf(),
+                  cumulativeAmount: datum.cumulativeAmount,
+                } as TradingRewardsDatum)
```

Comment:
Does this work if we make the anonymous function have a return type of TradingRewardsDatum instead of casting? I think it's generally safer that way. 
```
(datum): TradingRewardsDatum => ({
```